integration-test/test-pages/webcompat/pages/modify-localstorage.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Modify localStorage</title>
+    <link rel="stylesheet" href="../../shared/style.css">
+    <script>
+       window.localStorage.setItem('keyToBeDeleted', 'valueToBeDeleted')
+       window.localStorage.setItem('otherKey', 'valueToRemain')
+    </script>
+</head>
+<body>
+    <script src="../../shared/utils.js"></script>
+    <p><a href="../index.html">[WebCompat]</a></p>
+
+    <p>This page verifies that localStorage modifications work properly given the <a href="../config/modify-localstorage.json">config</a>. At this time, only deletion is supported.</p>
+
+    <script>
+        // eslint-disable-next-line no-undef
+        test('Only specified localStorage entry should be removed', async () => {
```

Comment:
Can we add a test for a key that is also isn't present but the code deletes?

Reply:
This what you meant? https://github.com/duckduckgo/privacy-configuration/pull/2283/files#diff-5312306cd0a26ef66eb3a99c46d298823c8a3419967021092d895bab48d2e2d1R72-R87 and line 28 below